CHANGELOG.md

```diff
@@ -9,3 +9,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+
+- Added Mean Reciprocal Rank metric
```

Comment:
Agreed

It's funny because some guy who wrote example for changelog thinks otherwise 😄 
![image](https://user-images.githubusercontent.com/36198818/219102138-a523f925-8afa-4bd8-90a7-6fd4792bdd0d.png)

Reply:
Hmm